pkg/ibmcsidriver/node.go

```diff
@@ -213,9 +213,15 @@ func (csiNS *CSINodeServer) NodeGetInfo(ctx context.Context, req *csi.NodeGetInf
 	ctxLogger, requestID := utils.GetContextLogger(ctx, false)
 	ctxLogger.Info("CSINodeServer-NodeGetInfo... ", zap.Reflect("Request", req))
 
+	nodeName := os.Getenv("KUBE_NODE_NAME")
```

Comment:
this is not required if csiNS.Metadata == nil

Reply:
hmm , seems we can shift the code. Done